### PR TITLE
Register xohcloud.is-a.dev

### PR DESCRIPTION
**Pull request by aysko-discord**
Register xohcloud.is-a.dev with TXT record pointing to xohcloud.fr.

### CODE REVIEW

File: domains/aysko.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "aysko-discord",
+           "email": "marckelyfc@icloud.com",
+           "discord": "370342658033319936"
+        },
+    
+        "record": {
+            "CNAME": "relaxed-sable-26f2ce.netlify.app."
```
Comment (CuteDog5695) [inline domains/aysko.json:9]: Site not found, also technically CNAME is invalid
```suggestion
            "CNAME": "relaxed-sable-26f2ce.netlify.app"
```

File: domains/ayskobtw.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "aysko-discord",
+        "email": "marckelyfc@icloud.com"
+    },
+    "record": {
+        "CNAME": "guns.lol/aysko"
```
Comment (CuteDog5695) [inline domains/ayskobtw.json:7]:
```suggestion
        "CNAME": "guns.lol"
```

File: domains/ayskobtww.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "aysko-discord",
+        "email": "marckelyfc@icloud.com"
+    },
+    "record": {
+        "URL": "guns.lol/aysko"
```
Comment (CuteDog5695) [inline domains/ayskobtww.json:7]: URL records need to have protocol
```suggestion
        "URL": "https://guns.lol/aysko"
```

File: domains/xohcloud.json [hunk 4]
```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "aysko-discord",
+           "email": "",
+           "discord": "370342658033319936",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.fPpjcDtP8RfinC1_HkkyFGtzfTs8OgRnfMQ5wMUuXvDmoNAyUttezE5AUeOGxJt28mr7LzQOIfQh4nNKLym6VZKAsvEzsgd0lWeEBO0Gd5zhPlH0JHpMHDshFPYNnM_HToZHW5UsvvqzdHBqj2cJr2WFnCPV730M-pShI1W9rzNKDo1RFXSQrYqXNPavLOiV7AlNytOIa-A7rjESGjOTl8WgArR3SjQe1zQo3JyR0-H-TOq19uM41VjrDFjZlbF3PVOr1AEpLYFKnmsbaOuz5F-sKJuXLJkf6b_Xn-WWNupbH4OyfmMm1zqi23f6_d4z9hAC88vfsxIoVNG8qDXRDA.RP6RuSkI9BBQ_QD2aoKF9g.vmdi2NcZ39RSAEriny6iOl9AzH5FKSM1NIFvyQSZcDIjOAQFzImEw7VNQ4x6goqyQOz2l9udaaQG4GWTnOzXn0LvSTYyozfJVqm1gHBYPtk.kZ9j9BMv20_-LYykrRLPHg"
+        },
+    
+        "record": {
+            "TXT": "xohcloud.fr"
```
Comment (CuteDog5695) [inline domains/xohcloud.json:10]: TXT doesn't do anything technically

**Review by iostpa — CHANGES_REQUESTED** [review]

**Comment by orxngc** [comment]
Closing as stale.

**Pull request closed without merging**